protobuf/io/casperlabs/ipc/ipc.proto

```diff
@@ -252,6 +252,7 @@ message ChainSpec {
         CostTable costs = 7;
         DeployConfig deploy_config = 8;
         HighwayConfig highway_config = 9;
+        repeated string state_infos = 15;
```

Comment:
Why 15, not 11?

Reply:
To prevent crashes when casper is added in the future.

Reply:
If some elements are added by upstream, anyway, there will be a conflict which we should resolve. We can handle them when it takes place and leaping off some range without comment could seem to be ambiguous.

Reply:
Do you want to write 11?
Or is it okay to just add a comment?

I don't think it's a big deal. I think it will be a problem because it will merge without crash when importing upstream.

Reply:
I agree with it's not a big deal. But, both modified file cannot be merged without conflict.
I think it is better to use 11, but leaving a comment is also acceptable.

Reply:
I think it will merge without conflict. So, with 11, the error will be off when proto generation, and if you write a high value, I think there will be no problem. And I think that changing the protocol number is not a good way to prepare for a soft fork.

But it's not a big deal, so I'll just apply it to 11.

Reply:
If they can be merged without conflict as you said, applying it to 11 seems a big deal.
But it will never happen. If you are still wondering, I recommend you to try in local.
(I've already tried)

Also, since we've not released our main-net yet, I guess there will be a no soft fork.

Reply:
In v0.18.0, 10 HighwayConfig came in, and I used state_info as 10. Both were added without conflict when rebase the process.

The number of soft forks I talked about is as follows. We use 11, and casperlabs add another protocol that uses 11. However, that part is assumed to be data that does not affect state or set values or other data. Even if some nodes do not support casper 11, there will be no problem with node operation.